daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -2438,9 +2456,17 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
           )
           inside(res) {
             case Success(
-                  Left(SErrorDamlException(IE.ContractKeyNotVisible(_, key, _, _, _)))
+                  Left(
+                    SErrorDamlException(
+                      IE.FailedAuthorization(
+                        _,
+                        FetchMissingAuthorization(`T`, _, stakeholders, authParties),
```

Comment:
As local contracts now always pass visibility checking, we now fail due to authorisation issues